### DISCUSSION TITLE
Ontology Access Kit (OAK) tutorial

### DISCUSSION THREAD

**Discussion by shawntanzk** [jamesaoverton/obook#167] [Tutorial requests]
I know this is super premature as this is pretty new and isn't in ODK till next release, but I see that @gouttegd just merged into ODK and im pretty excited about it, so putting it here now heh.

**Comment by cmungall** [comment]
@shawntanzk I hope you can make it to the introductory OAK workshop next week https://docs.google.com/document/d/17wZwoipL6l92kltCaWjMwUMb-OIz-ugehEEfR2SrdAA/edit#heading=h.kytdhk7097ls

**Comment by matentzn** [comment]
This has happened https://www.youtube.com/watch?v=zO6tS7gZtvw
